src/strategies/squadz-power/index.ts

```diff
@@ -0,0 +1,48 @@
+import { multicall } from '../../utils';
+
+export const author = 'EzraWeller';
+export const version = '0.0.1';
+
+const engineAddresses: { [network: string]: string } = {
+  '5': "0x7beaa4e60e0faab603e99813f0f2330704b53086",
+  '80001': "0x39235b78626d8fa4ef6a81ba5616c58708ba4ea5",
+  '137': "0xb4a1a96ffa514b295b9a0de127288ec7d09e4e7c",
+  '4': "0xbeea7483aef24502a27eb7a35aad55280f8e2ebc"
+}
+
+const engineAbi = [
+  'function getMemberInfo(address, uint256, address) view returns (uint256, uint256, uint256, bool, bool, uint256, uint256)'
+]
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  if (Object.keys(engineAddresses).includes(network))
+    console.warn('Invalid network:', network, 'not in', Object.keys(engineAddresses));
```

Comment:
This shows a warning always:
```
console.warn
    Invalid network: 5 not in [ '4', '5', '137', '80001' ]

      17 |     const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
      18 |     if (Object.keys(engineAddresses).includes(network))
    > 19 |         console.warn('Invalid network:', network, 'not in', Object.keys(engineAddresses));
         |                 ^
      20 |     const engineAddress = engineAddresses[network];
      21 |     const forkNumber = options.forkNumber ?? 0;
      22 |     const response = await (0, utils_1.multicall)(network, provider, engineAbi, addresses.map(member => [
      ```
      maybe there is should be not includes, but we don;'t really need this :) better to remove,

Reply:
Would be fine to remove, but it would nice to have some indicator somewhere of which networks are supported. Is there a better place for that?